local/local.go

```diff
@@ -71,28 +76,107 @@ func FromBaseImage(imageName string) ImageOption {
 	}
 }
 
-func NewImage(repoName string, dockerClient client.CommonAPIClient, ops ...ImageOption) (*Image, error) {
+func WithPlatform(platform imgutil.Platform) InitialImageOption {
```

Comment:
What do you think about `WithDefaultPlatform` or `WithPlatformDefault`? I was thinking that the `Default` might convey this wouldn't be used in the scenario where you are using a previous image.

But maybe after Emily's comments/suggestions - this is no longer needed.

Reply:
I like this renaming suggestion, assuming we keep the behavior as it is. `WithDefaultPlatform` feels best to me, as it follows proposition-adjective-noun pattern of `FromBaseImage`/`FromPreviousImage`.

Reply:
WithDefaultPlatform feels like a good fit for the current behavior.